openlibrary/templates/home/index.html

```diff
@@ -15,11 +15,17 @@
 $add_metatag(name="twitter:image:alt", content="Open Library Logo")
 $add_metatag(name="twitter:card", content="homepage_summary")
 
+$code:
+  FEB_READS = 'key:(/works/OL5099301W OR /works/OL20900813W OR /works/OL19974130W OR /works/OL21376789W OR /works/OL21179764W OR /works/OL491675W OR /works/OL20848742W OR /works/OL5730175W OR /works/OL3496457W OR /works/OL18023357W OR /works/OL16910779W OR /works/OL16662711W OR /works/OL20857543W OR /works/OL20632743W OR /works/OL19729253W OR /works/OL17314666W OR /works/OL20858133W OR /works/OL21672239W OR /works/OL19757671W OR /works/OL1928280W OR /works/OL17874444W OR /works/OL20783908W OR /works/OL18335154W OR /works/OL4114499W OR /works/OL20872535W OR /works/OL20153225W OR /works/OL2639036W OR /works/OL20207960W OR /works/OL20544657W OR /works/OL20854371W OR /works/OL17869588W OR /works/OL440489W OR /works/OL3191599W OR /works/OL8275668W OR /works/OL17102898W OR /works/OL17721078W OR /works/OL20007451W OR /works/OL21180618W OR /works/OL20848626W OR /works/OL19972590W OR /works/OL1486478W OR /works/OL11952386W OR /works/OL19994722W OR /works/OL7870775W OR /works/OL20646859W OR /works/OL4648142W OR /works/OL11999803W OR /works/OL16515210W OR /works/OL15279153W OR /works/OL20849927W OR /works/OL21179974W OR /works/OL3521634W OR /works/OL16097454W OR /works/OL9371452W OR /works/OL4802599W OR /works/OL20030448W OR /works/OL3296483W OR /works/OL17371695W OR /works/OL22062636W OR /works/OL8990067W OR /works/OL7213898W OR /works/OL19708155W OR /works/OL17358927W OR /works/OL20853865W OR /works/OL20548582W OR /works/OL16876826W OR /works/OL13655581W OR /works/OL4370955W OR /works/OL2977588W OR /works/OL21078916W OR /works/OL20133929W OR /works/OL16617686W OR /works/OL5706069W OR /works/OL15691480W OR /works/OL17755326W OR /works/OL15933199W OR /works/OL17893385W OR /works/OL17411311W OR /works/OL17933439W OR /works/OL4622460W OR /works/OL16025481W OR /works/OL17640891W OR /works/OL20053695W OR /works/OL19996443W OR /works/OL19854767W OR /works/OL4582875W OR /works/OL15285884W OR /works/OL17340085W OR /works/OL20022939W OR /works/OL9242636W OR /works/OL519219W OR /works/OL17566265W OR /works/OL22074932W OR /works/OL20907796W OR /works/OL20879942W OR /works/OL22059158W OR /works/OL17430949W OR /works/OL20935567W OR /works/OL18646141W OR /works/OL19646242W)'
```

Comment:
Was still getting a few "Not in Library", so updated my little JS snippet to avoid those.

```suggestion
  FEB_READS = 'key:(/works/OL18181363W OR /works/OL3481095W OR /works/OL4360244W OR /works/OL20017931W OR /works/OL20615204W OR /works/OL2363176W OR /works/OL17869588W OR /works/OL17784026W OR /works/OL21179764W OR /works/OL8870595W OR /works/OL21054973W OR /works/OL21673730W OR /works/OL20548582W OR /works/OL15279153W OR /works/OL19992836W OR /works/OL15691480W OR /works/OL16305795W OR /works/OL19923407W OR /works/OL16529029W OR /works/OL9242636W OR /works/OL17529769W OR /works/OL3345332W OR /works/OL20013209W OR /works/OL20015483W OR /works/OL19987474W OR /works/OL19992114W OR /works/OL17893900W OR /works/OL18435803W OR /works/OL17314666W OR /works/OL17358927W OR /works/OL15933199W OR /works/OL17858931W OR /works/OL18187603W OR /works/OL16853133W OR /works/OL16894393W OR /works/OL19976062W OR /works/OL20037832W OR /works/OL16885033W OR /works/OL19708155W OR /works/OL17921756W OR /works/OL21037237W OR /works/OL17786027W OR /works/OL17345141W OR /works/OL21294275W OR /works/OL9582417W OR /works/OL9357555W OR /works/OL20907853W OR /works/OL20005568W OR /works/OL3296483W OR /works/OL11983310W OR /works/OL7159886W OR /works/OL1662667W OR /works/OL19990553W OR /works/OL15285884W OR /works/OL6888879W OR /works/OL17900435W OR /works/OL5706069W OR /works/OL2977589W OR /works/OL1593701W OR /works/OL16451688W OR /works/OL16910779W OR /works/OL18215336W OR /works/OL17371695W OR /works/OL3521634W OR /works/OL17355199W OR /works/OL5739152W OR /works/OL20016962W OR /works/OL3191599W OR /works/OL20896695W OR /works/OL19752490W OR /works/OL18335154W OR /works/OL4582875W OR /works/OL16515210W OR /works/OL16868407W OR /works/OL3459949W OR /works/OL16025481W OR /works/OL1928280W OR /works/OL6208302W OR /works/OL17566265W OR /works/OL20652811W OR /works/OL22059158W OR /works/OL4370955W OR /works/OL19998526W OR /works/OL6218060W OR /works/OL16813953W OR /works/OL21179974W OR /works/OL7213898W OR /works/OL17872185W OR /works/OL17340085W OR /works/OL21584979W OR /works/OL21078916W OR /works/OL158519W OR /works/OL4114499W OR /works/OL19638041W OR /works/OL16844793W OR /works/OL20940485W OR /works/OL17392121W OR /works/OL20030448W OR /works/OL15920474W OR /works/OL20544657W)'
```